src/index.ts

```diff
@@ -276,6 +279,34 @@ class Monika extends Command {
           em.emit('SANITIZED_CONFIG', sanitizedProbe)
         }
 
+        // run TLS checker
+        if (config?.certificate && config.certificate?.domains.length > 0) {
+          config.certificate?.domains.forEach(async (domain) => {
+            // TODO: Remove probe below
+            // probe is used because probe detail is needed to save the notification log
+            const probe = sanitizedProbe[0]
+            // check TLS when Monika starts
+            this.checkTLSAndSaveNotifIfFail(
+              domain,
+              config.certificate?.reminder ?? 30,
+              probe,
+              config?.notifications
+            ).catch((error) => log.error(error.message))
```

Comment:
`checkTLSAndSaveNotifIfFail` will not throwing any error. This line is (probably) dead code.

Reply:
Thank you, nice catch. I fixed it here [
1088244](https://github.com/hyperjumptech/monika/pull/315/commits/10882445203b6203a1c603de4021f4da25e77a54).